BUILDING.md

```diff
@@ -1,255 +1,49 @@
-# Building VTR #
+# Building VTR
 
 
-## Overview ##
+## Setting up Your Environment
 
-VTR uses [CMake](https://cmake.org) as it's build system.
 
-CMake provides a portable cross-platform build systems with many useful features.
+VTR requires several system packages.  From the top-level directory, run the following script to install the required packages on a modern Debian or Ubuntu system:
 
-## Tested Compilers ##
-VTR requires a C++-14 compliant compiler.
-It is tested against the default compilers of all Debian and Ubuntu releases within their standard support lifetime. Currently, those are the following:
+    ./install_apt_packages.sh
 
-* GCC/G++: 7, 8, 9, 10, 11
-* Clang/Clang++: 6, 7, 10
 
-Other compilers may work but are untested (your milage may vary).
+You will also need several Python packages.  You can optionally install and activate a Python virtual environment so that you do not need to modify your system Python installation:
 
-## Unix-like ##
-For unix-like systems we provide a wrapper Makefile which supports the traditional `make` and `make clean` commands, but calls CMake behind the scenes.
+    make env
+    source .venv/bin/activate
 
-### Dependencies ###
+Then to install the Python packages:
 
-For the basic tools you need:
- * Bison & Flex
- * cmake, make
- * A modern C++ compiler supporting C++14 (such as GCC >= 4.9 or clang >= 3.6)
+    pip install -r requirements.txt
 
-For the VPR GUI you need:
- * Cairo
- * FreeType
- * Xft (libXft + libX11)
- * fontconfig
- * libgtk-3-dev
+**Note:** If you chose to install the Python virtual environment, you will need to remember to activate it on any new terminal window you use, before you can run the VTR flow or regressions tests.
```

Comment:
Might be good to give the exact command to run to activate the environment here, maybe in brackets.  Especially useful if the activate gets moved into the make env command, so you only have to run this command in other windows.

Reply:
Fixed.